ts/types/globals.d.ts

```diff
@@ -1 +1,3 @@
 declare let ige: IgeEngine;
+declare let openChatBubble: any;
+declare let _: any;
```

Comment:
should we add lodash types?

Reply:
we might if it is easy to add, pixi types are basically impossible to make work with our current ts setup for instance